configure

```diff
@@ -1064,7 +1064,7 @@ fi
 # For the Pervasives module
 
 if sh ./hasgot2 -i math.h $mathlib expm1 log1p hypot copysign; then
```

Comment:
You should include `fma` in the list of functions tested with hasgot2.

Reply:
Done